server/index.js

```diff
@@ -13,13 +13,15 @@ app.listen(port, function() {
 });
 
 app.get('/hotel/:hotel', (req, res) => {
-  console.log('in');
-  console.log('hotel', req.params.hotel);
-  let hotel = req.params.hotel === 'root'? 'hotel0': req.params.hotel;
+  let hotel = req.params.hotel === 'global'? 'hotel0': req.params.hotel;
   let id = hotel.slice(5);
-  db.getReviewsByHotel(id)
+  console.log(`in hotel ${id}`);
+  db.getHotelReviews(id)
     .then(result => {
       res.send(result);
+    })
+    .catch((err) => {
+      console.log(err);

```

Comment:
### Error Specificity
This catch block will spit out your error as a normal log. I'd recommend switching to...
`console.error('ERROR: short description of error', err);`
This will pop the error into your console highlighted red, and the message will tell you exactly where the problem is without having to rely on traces.

(I sometimes drop the ball on this, too. Trying to get better about it.)